dkist/io/asdf/resources/manifests/dkist-wcs-1.4.0.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.1
+---
+id: asdf://dkist.nso.edu/manifests/dkist-wcs-1.4.0
+extension_uri: asdf://dkist.nso.edu/dkist/extensions/dkist-wcs-1.4.0
+asdf_standard_requirement:
+  gte: 1.6.0
+
+title: DKIST WCS extension
+description: ASDF schemas and tags for models and WCS related classes.
+
+tags:
+  # the tag version does not match the schema version
+  - schema_uri: "asdf://dkist.nso.edu/schemas/varying_celestial_transform-1.2.0"
+    tag_uri: "asdf://dkist.nso.edu/tags/varying_celestial_transform-1.3.0"
```

Comment:
Is there a reason these have different version numbers?

Reply:
Presumably because at some point in the past we bumped the tag and not the schema.